### PR TITLE
del metadata sync && del duplicate metrics

### PR DESCRIPTION
**Pull request by zwwhdls**
### Ⅰ. Describe what this PR does
1. delete metadata sync of juicefs. At present, the metadata sync of juicefs is realized through `ls -lR`. When the size of files is too large, the result returned by this command will cause exec to occupy a large amount of memory. And metadata sync is not necessary for juicefs.
2. duplicate `getMetrics` in juicefs runtime controller, make it only get once.

### Ⅱ. Does this pull request fix one issue?

fix https://github.com/fluid-cloudnative/fluid/issues/3381

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/ddc/juicefs/operations/base.go [hunk 1]
```diff
@@ -319,17 +319,13 @@ func (j JuiceFileUtils) exec(command []string, verbose bool) (stdout string, std
 }
 
 // execWithoutTimeout
-func (j JuiceFileUtils) execWithoutTimeout(command []string, verbose bool) (stdout string, stderr string, err error) {
+func (j JuiceFileUtils) execWithoutTimeout(command []string) (stdout string, stderr string, err error) {
 	stdout, stderr, err = kubeclient.ExecCommandInContainer(j.podName, j.container, j.namespace, command)
 	if err != nil {
-		j.log.Info("Stdout", "Command", command, "Stdout", stdout)
+		j.log.V(1).Info("Stdout", "Command", command, "Stdout", stdout)
```
Comment (cheyang) [inline pkg/ddc/juicefs/operations/base.go:325]: I think it's not user friendly when debugging issue.

Comment (zwwhdls) [reply pkg/ddc/juicefs/operations/base.go:325]: `info` directly ?

File: pkg/ddc/juicefs/operations/base.go [hunk 2]
```diff
@@ -319,17 +319,13 @@ func (j JuiceFileUtils) exec(command []string, verbose bool) (stdout string, std
 }
 
 // execWithoutTimeout
-func (j JuiceFileUtils) execWithoutTimeout(command []string, verbose bool) (stdout string, stderr string, err error) {
+func (j JuiceFileUtils) execWithoutTimeout(command []string) (stdout string, stderr string, err error) {
```
Comment (cheyang) [inline pkg/ddc/juicefs/operations/base.go:322]: I suggest making the verbose configurable for future debugging.

**Pull request merged**